tdee_macro_calc/main.py

```diff
@@ -6,16 +6,17 @@
 
 c = Console()
 
-def check_data_file() -> dict:
+HOME = Path.home()
```

Comment:
Oh I see what's going on here, this way we can pass in the `tmp_path` in the test. 

In general, as far as conventions go, is it better to make this a global variable anyway?